data_architect_misc/data_transformer/constants/comp_harm_constants.py

```diff
@@ -188,13 +197,17 @@
     "(?i).*Investment.*": "Other",
     "(?i).*Land.*": "Other",
     "(?i).*Materials.*": "Other",
+    "(?i).*Medicated.*": "Other",
     "(?i).*Miscellaneous.*": "Other",
     "(?i).*Office.*": "Other",
     "(?i).*Petroleum.*": "Other",
+    "(?i).*Pesticides.*": "Other",
     "(?i).*PHARMACY.*": "Other",
     "(?i).*Retail.*": "Other",
+    "(?i).*Remedy.*":"Other",
```

Comment:
could be 'hair remedy' (conditioner). remedy means fixing. it could mean fixing your hair, your skin.. please remove this. what you should do is to get the exact or closest product name (like this '(?i).*headache.*':), and add it for specific to malaysia file like what we did in INDIA_SPECIFIC_CATEGORY_MAPPINGS (not in the standard category mapping).